### PR TITLE
Add logzio storage plugin

### PR DESCRIPTION
**Pull request by yyyogev**
## Which problem is this PR solving?
Missing new storage plugin - logz.io
## Short description of the changes
Add a link to jaeger-logzio repo

### CODE REVIEW

File: content/docs/next-release/deployment.md [hunk 1]
```diff
@@ -398,6 +398,7 @@ Jaeger supports gRPC based storage plugins. For more information refer to [jaege
 
 Available plugins:
 
+* [Logz.io](https://github.com/logzio/jaeger-logzio)
```
Comment (yurishkuro) [inline content/docs/next-release/deployment.md]: - please move into second place
- this is not an open-source backend, correct? Please clarify that this is commercially hosted storage

Comment (yyyogev) [reply content/docs/next-release/deployment.md]: @yurishkuro is this clear enough?

File: content/docs/next-release/features.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Jaeger backend, Web UI, and instrumentation libraries have been designed from gr
 ## Multiple storage backends
 
 Jaeger supports two popular open source NoSQL databases as trace storage backends: Cassandra 3.4+ and Elasticsearch 5.x/6.x/7.x.
-There are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB. Jaeger also ships
+There are ongoing community experiments using other databases, such as Logz.io ,ScyllaDB, InfluxDB, Amazon DynamoDB. Jaeger also ships
```
Comment (yurishkuro) [inline content/docs/next-release/features.md]:
```suggestion
There are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB, Logz.io. Jaeger also ships
```

File: content/docs/next-release/deployment.md [hunk 3]
```diff
@@ -399,6 +399,7 @@ Jaeger supports gRPC based storage plugins. For more information refer to [jaege
 Available plugins:
 
 * [InfluxDB](https://github.com/influxdata/jaeger-influxdb/)
+* [Logz.io](https://github.com/logzio/jaeger-logzio) - Secure & Scalable Log Management with Cloud-Based ELK.
```
Comment (yurishkuro) [inline content/docs/next-release/deployment.md]:
```suggestion
* [Logz.io](https://github.com/logzio/jaeger-logzio) - secure, scalable, managed, cloud-based ELK storage.
```

**Comment by yyyogev** [comment]
@yurishkuro Thanks a lot!
We'd really appreciate it if you will also add it in the [storages thread](https://github.com/jaegertracing/jaeger/issues/638)

**Comment by yurishkuro** [comment]
done

**Pull request merged**